test/e2e/multivc_test.go

```diff
@@ -33,6 +33,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	infrav1 "sigs.k8s.io/cluster-api-provider-vsphere/apis/v1beta1"
+	. "sigs.k8s.io/cluster-api-provider-vsphere/test/e2e/helper"
```

Comment:
```suggestion
```

Reply:
Does this test also need a CP IP?

Reply:
This tests can't run in upstream CI which is why I did not use the IPAM helper here.

This file uses the `Byf` function though which I moved to the seperated package.

Reply:
Let's assume we would be able to run this test, would it then make sense to use the func? I think then I would be in favor of using the helper here as well for consistency (I assume it's still a no-op if the e2e config is not set)